p/themes/base-theme/template.css

```diff
@@ -772,6 +772,73 @@ br {
 	vertical-align: middle;
 }
 
+/*=== Popup */
+#popup {
+	display: none;
+	position: fixed;
+	z-index: 1;
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: rgb(0,0,0);
+	background-color: rgba(0,0,0,0.4);
```

Comment:
Since the idea here is something a lot closer to "gray" than to "black", the fallback should probably be something more like `#eee` than like `#000`?

Reply:
Yes, you are right. Fixed.